_posts/platform/app/2000-01-01-run-scheduled-tasks.md

```diff
@@ -47,3 +58,35 @@ scalingo --app my-app cron-tasks
 | */10 * * * * echo biniou |      |
 +--------------------------+------+
 ```
+
+## Delete Configured Cron Tasks
+
+In order to delete a cron tasks, you must delete the associated entry in the `cron.json`.
+If there is no more tasks, or if you want to remove all tasks, you can also remove the file.
+
+The cron tasks configuration will be updated when redeploying the application.
+
+## FAQ
+
+### How To Enable Cron Tasks?
+
+The cron feature is enabled by default.
+
+### How To Disable Cron Tasks?
+
+You may want to disable automatic cron tasks execution, for example for [review apps]({% post_url
+platform/app/2000-01-01-review-apps %}) or staging environments.
+
+TBD
+
+### How Precise Is The Cron Feature?
+
+Since the cron feature needs to start a new container to execute the task, the precision will depend on the
+deployment time.
```

Comment:
This is not true, it does not depend on the deployment time. We don't trigger a new deployment every time we execute a task. It depends on the image size as we need to pull the application image which may take a few minutes